app/src/main/res/layout/activity_main.xml

```diff
@@ -4,15 +4,26 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".MainActivity">
+    android:background="@drawable/wallpaper_main_screen"
+    tools:context=".activities.MainActivity">
 
     <TextView
+        android:id="@+id/text_main_activity_title"
+        style="@style/TitleStyleMainActivity"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/text_main_activity_margin_top"
+        android:text="@string/main_activity_text_title"
+        app:layout_constraintTop_toTopOf="parent" />
+    <Button
+        android:id="@+id/button_main_activity_select_parking_space"
+        android:backgroundTint="@color/button_main_activity_cream"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Hello World!"
+        android:text="@string/main_activity_button_set_size"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintLeft_toLeftOf="parent"
         app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintTop_toBottomOf="@+id/text_main_activity_title" />
```

Comment:
pls remove "+" , because you can access for all the ids in the layouts, but you need to access only in this layout

Reply:
done